test/imgcomp.jl

```diff
@@ -99,7 +99,7 @@ function image_comparison_facts(pkg::Symbol;
   for i in 1:length(Plots._examples)
     i in skip && continue
     if only == nothing || i in only
-      @fact image_comparison_tests(pkg, i, debug=debug, sigma=sigma, eps=eps) |> success --> true
+      @fact @eval(image_comparison_tests(Symbol(String(Symbol($pkg))[7:end]), $i, debug=$debug, sigma=$sigma, eps=$eps)) |> success --> true
```

Comment:
I don't really like `Symbol(String(Symbol($pkg))[7:end])` here. Maybe someone has a more elegant solution. `$pkg` returns e.g. for GR the backend `Plots.gr`.